versions/development/SPEC.md

```diff
@@ -2576,14 +2652,15 @@ workflow wf {
 
   # Calls my_task with one required input - it is okay to not
   # specify a value for my_task.opt_string since it is optional.
-  call my_task { input: num = i }
+  call my_task { input: num = i env: NUM = i }
```

Comment:
I don't super love this. I think all tool interactions should be handled by the tool wrapper and not be the caller.

Reply:
I second this.

Reply:
That should definitely be removed, I think its outdated from the previous iteration. I will update the PR